cartridges/app_custom_page_designer/cartridge/templates/default/experience/components/commerce_assets/categoryTile.isml

```diff
@@ -0,0 +1,58 @@
+<isscript>
+    var assets = require('*/cartridge/scripts/assets.js');
+    assets.addCss('/css/experience/components/commerceAssets/popularCategory.css');
+</isscript>
+
+<isif condition="${pdict.category.ctaPosition == "top"}">
+    <div class="popular-category-link">
+        <a href="${pdict.category.url}" class="popular-category-label-a popular-cat-link">${pdict.category.ctaTitle}</a>
+    </div>
+</isif>
+
+<div class="mt-2">
+    <div class="image-cropper">
+        <a href="${pdict.category.url}">
+            <isif condition="${pdict.image}">
+               <style>
+                    .image-wrapper-${pdict.category.compID} span {
+                        background-image: url('${pdict.image.src.mobile}');
+                        background-repeat: no-repeat;
+                        background-position: center center; 
+                    }
+                    @media (min-width: 768px) {
+                        .image-wrapper-${pdict.category.compID} span {
+                            background-image: url('${pdict.image.src.tablet}');
+                        }
+                    }
+                    @media (min-width: 992px) {
+                        .image-wrapper-${pdict.category.compID} span {
+                            background-image: url('${pdict.image.src.desktop}');
+                        }
+                    }
+                </style>
+                <div class="image-wrapper-${pdict.category.compID} image-wrapper">
+                    <span class="background-image" role="img" aria-label="${pdict.category.description}"> </span>
+                </div> 
+            <iselse>
+                <div class="image-wrapper"
+                    <isif condition="${pdict.category.imageURL}">
+                        style="background-image: url(${pdict.category.imageURL}); background-repeat: no-repeat;
+                        background-position: center center; background-size: cover;"
+                    </isif>
+                 >
+                <span class="background-image" role="img" aria-label="${pdict.category.name}"></span>
+            </div>
+            </isif>
+        </a>
+    </div>
+
+    <div class="text-center my-1">
+        <span>${pdict.category.description}</span>
+    </div>
+</div>
+
+<isif condition="${pdict.category.ctaPosition == "bottom"}">
```

Comment:
I would suggest instead to use class, position: absolute and fine tune the position of the CTA only with CSS. Think that in future might have more options than top/bottom, hence current approach is not scalable.